README.md

```diff
@@ -1,10 +1,32 @@
-# plantalytics-hardware
+# Plantalytics Microcontrollers
 
-The Plantalytics network relies on a series of nodes (Motenino USB RFm95 with LoRa 915mhz Transceiver).
-
-------------------------------------------------------------------------------------------------
 ## License
 
 Copyright (c) 2016 Sapphire Becker, Katy Brimm, Scott Ewing, Matt Fraser, Kelly Ledford, Michael Limb, Steven Ngo, Eric Turley.
 
 This project is licensed under the MIT License. Please see the file LICENSE in this distribution for license terms.
+
+## How to Use
+
+Get your hardware:
+* **Micro controller** (Motenino USB RFM95 with LORA 915mhz Trasnsceiver)
+
+Install the Arduino IDE:
+* **Arduino IDE** (version 1.6.9 or greater)
+  * https://www.arduino.cc/en/Main/Software
+* Under tools tab, make sure the board is "Arduino/Genuino Uno".
+  
+Install the following in your Arduino/libraries directory:
+* **IMIC Master Library** 
+  * https://github.com/matthijskooijman/arduino-lmic
+* **Radiohead Library** 
+  * http://lowpowerlab.com/RadioHead_LowPowerLab.zip
+* **DHT Sensor Library**
+  * https://github.com/adafruit/DHT-sensor-library
+
+Plug the micro controller into the computer, and set the device address at line 48. When compiled, the program will be pushed to the micro-controller and execute immediately!
+
+## Note:
+
```

Comment:
Maybe note the FINAL FINAL version for our NODE code is now called node controller?